dipy/tracking/local_tracking.py

```diff
@@ -169,17 +174,19 @@ def _generate_tractogram(self):
 
                 # move to the next streamline if only the seed position
                 # and not return all
-                if len(streamline) > 1 or self.return_all:
-                    if self.save_seeds:
-                        yield streamline, s
-                    else:
-                        yield streamline
+                len_sl = len(streamline)
+                if len_sl >= self.min_length and len_sl <= self.max_length \
```

Comment:
same here, As you can see in this link: https://github.com/dipy/dipy/blob/eede25d73dcff7fff758d64dbdb829533de6a05a/dipy/tracking/local_tracking.py#L172

it was `> minlen`  with minlen default to 1.  I would recommend to keep this condition and default value to avoid breaking previous code